low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -18,14 +18,13 @@
 
 import time
 import re
```

Comment:
Unused import.

Reply:
`re` is no longer required as `get_node_details` function is moved to resource_map.py.

Reply:
removed.